scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -627,7 +627,9 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
                     outputDir,
                     "--configuration",
                     configuration,
-                    Path.Combine(c.BuildContext.BuildDirectory, "src", project))
+                    Path.Combine(c.BuildContext.BuildDirectory, "src", project),
+                    "--version-suffix",
+                    buildVersion.VersionSuffix)
```

Comment:
Look @line 622. it suggests we add this. Since we need for version suffix to be understood, and it seemed not to be, added this optimistically.